pkg/controllers/binding/common.go

```diff
@@ -101,6 +93,7 @@ func ensureWork(
 		}
 
 		workLabel := mergeLabel(clonedWorkload, workNamespace, binding, scope)
+		hasScheduledReplica, desireReplicaInfos := getReplicaInfos(targetClusters)
```

Comment:
Why move it into the loop, it does not need to be calculated every time

Reply:
oops, big mistake. fixed.